pinot-tools/src/main/java/org/apache/pinot/tools/RealtimeQuickStart.java

```diff
@@ -50,6 +51,8 @@ public static void main(String[] args)
   protected Map<String, Object> getConfigOverrides() {
     Map<String, Object> configOverrides = new HashMap<>();
     configOverrides.put(CommonConstants.Server.CONFIG_OF_ENABLE_THREAD_CPU_TIME_MEASUREMENT, true);
+    configOverrides.put(SegmentCompletionConfig.FSM_SCHEME + "pauseless",
```

Comment:
Let's not Add this here. Instead create a new quickstart called `PauselessRealtimeQuickstart` where pausless is also enabled for all tables

We can do that in a seperate PR